### ISSUE TITLE
"Base table or view not found" after composer install

### ISSUE THREAD

**Issue by incognitae** [Dynamedia/wn-posts-plugin#1]
hi Rob
thank you for your work.

But I encounter an error in the backend after an installation via composer
`SQLSTATE[42S02]: Base table or view not found: 1146 Table xxxx.dynamedia_posts_profiles' doesn't exist (SQL: select * from `dynamedia_posts_profiles` where `dynamedia_posts_profiles`.`user_id` = 1 and `dynamedia_posts_profiles`.`user_id` is not null limit 1)`

I have executed :
`composer require dynamedia/wn-posts-plugin`
`php artisan winter:up`

I only got the migrations of the dependencies but not of wn-posts-plugin
and so the error message

wn-posts-plugin 1.0.3
wintercms 1.1.6

did I miss something?
Thanks for your advice

**Comment by robballantyne** [comment]
Hi @incognitae 

As far as I can see you have not missed anything so I will try and replicate this although so far I have not encountered this problem.

Was this a fresh install of Winter with the default migrations already run?

The versions you have quoted are correct and should install without issue but I will further test and update ASAP.

Thanks for your report, I really appreciate you pointing this out.

Rob

**Comment by incognitae** [comment]
thank you for your answer Rob.
here is a sample of my composer.json with plugins already installed and working very well.
All migrations are running smoothly.
```
  "require": {
    "php": ">=7.2.9",
    "winter/storm": "dev-develop as 1.1.999",
    "winter/wn-system-module": "dev-develop as 1.1.999",
    "winter/wn-backend-module": "dev-develop as 1.1.999",
    "winter/wn-cms-module": "dev-develop as 1.1.999",
    "laravel/framework": "~6.0",
    "wikimedia/composer-merge-plugin": "~2.0.1",
    "winter/wn-pages-plugin": "^2.0.0",
    "winter/wn-location-plugin": "^v2.0.0",
    "winter/wn-mailchimp-plugin": "^v2.0.0",
    "winter/wn-user-plugin": "^v2.0.0",
    "vdlp/oc-redirect-plugin": "^2.5.5",
    "offline/responsiveimages-plugin": "v2.3.12",
    "offline/oc-site-search-plugin": "^v1.5.6",
    "toughdeveloper/imageresizer": "^v1.4.0",
    "martin/wn-forms-plugin": "^2.0"
  },
  "require-dev": {
    "phpunit/phpunit": "^8.4|^9.3.3",
    "mockery/mockery": "~1.3.3|^1.4.2",
    "fakerphp/faker": "~1.9",
    "squizlabs/php_codesniffer": "3.*",
    "php-parallel-lint/php-parallel-lint": "^1.0",
    "dms/phpunit-arraysubset-asserts": "^0.1.0|^0.2.1",
    "winter/wn-builder-plugin": "^v2.0.1"
  },
```

I also tested
`php artisan plugin:refresh Dynamedia.Posts`
with no effect.

I'll try a few other things tomorrow.
Thanks again.

**Comment by robballantyne** [comment]
This appears to be an issue with the version.yaml. Version keys are not being parsed as expected with php 7.  I'm working on a fix and will push it out asap. (v1.0.4 becomes  "1.0.4")

Thanks for your patience!

Rob

**Comment by incognitae** [comment]
on the way to the solution ;-)

Don't put pressure on yourself for me.
I can wait without any problem. Thanks for the responsiveness in any case.

**Comment by robballantyne** [comment]
@incognitae this issue should now be fixed in version 1.0.6. 

I have restructured the updates directory, structured the version keys as "x.x.x" rather than the "vx.x.x" supported by October CMS with php 7+.

I have also added some logic to avoid breaking any existing installs as well as rewriting the plugin version history to avoid duplication.

I hope this does fix the issue for you but please let me know if you have any more difficulty.

Thanks
Rob
